scipy/spatial/transform/tests/test_rigid_transform.py

```diff
@@ -10,19 +10,26 @@
     xp_vector_norm,
     is_numpy,
     xp_assert_close,
+    make_xp_pytest_marks,
+    make_xp_test_case,
+    xp_assert_equal,
+    xp_promote
 )
 import scipy._lib.array_api_extra as xpx
 
+lazy_xp_modules = [RigidTransform, normalize_dual_quaternion]
```

Comment:
```suggestion
lazy_xp_modules = [RigidTransform]
```
functions imported in the local namespace are automatically caught.

Reply:
A bit too much magical stuff is happening implicitly for my taste. But I guess this is also a pytest thing. The fact that `xp` is never declared in this file bugs me as well.